native/packages/react-native-bpk-component-text/stories.js

```diff
@@ -18,66 +18,88 @@
 
 /* @flow */
 
+/* eslint-disable */
```

Comment:
remove this once the sketch comparison images are gone

Reply:
+1 - we should probably lose this whole story, unless you think it'll continue providing value?